contribute.md

```diff
@@ -1,18 +1,43 @@
 ---
 title: How to Contribute
 ---
+# Contributing to ACN-syllabus
 
-Hey there. I hear you want to contribute. In this document we'll breeze through all the conventions in place. A lot of this stuff is really friggin important. We have to be very strict on the structure of this repo otherwise Tilde wont be able to consume it.
+Ready to contribute? We'd love to have you on board. In this document we'll breeze through all the conventions in place. A lot of this stuff is really friggin important. We have to be very strict on the structure of this repo otherwise Tilde wont be able to consume it. Here are some guidelines we'd like you to follow so that everything can go smoothly, Ok. Let's get cracking!
 
-Ok. Let's get cracking!
+-   [Choosing something to work on](#issues)
+-   [Submitting a PR](#pr)
+-   [Directory structure](#ds)
+-   [Naming conventions](#nc)
+-   [Static resources](#sr)
+-   [A note on headings](#headings)
+-   [Syllabus directory](#syllabus)
+-   [Understanding the frontmatter](#frontmatter)
+-   [Keys and values](#keys)
+-   [Flavours](#flavours)
+-   [Topic specific stuff](#topic)
+-   [Project specific goodies](#goodies)
+-   [Depricated and unimportant](#depricated)
 
-## Some guidelines
+
+## <a name="issues"></a> Choosing something to work on
+
+The first thing you should do is go to our [issues](https://github.com/Umuzi-org/ACN-syllabus/issues ) and look for something you might be interested in working on. The issues have what we call labels that help us distinguish between available and claimed issues, issues that are simple or hard and issues for hactoberfest, it is important to look at those labels before claiming and issue, labels an issue might have are:
+- status/help wanted
+- status/issue-claimed
+- good-first-issue
+- hactoberfest and 
+-experienced devs only
+
+if an issue has the `status/issue-claimed` label please skip it and if you a beginner it is advisable to start with the issues that have the `good-first-issue` label. You can assign yourself an issue by commenting your name on the comment section inside the issue, if you feel you need help with choosing an issue or you unsure of anything you can join our [discord channel](https://discord.com/channels/758708091583332382/762669571852468254) and you will get all the help you need. 
```

Comment:
Please change the discord link to an invite link not a channel link. You can use https://discord.gg/3jRwWPm